libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -442,6 +442,7 @@ void RC_Channel::init_aux_function(const aux_func_t ch_option, const aux_switch_
     case CLEAR_WP:
     case COMPASS_LEARN:
     case LANDING_GEAR:
+    case MOTOR_ESTOP:
```

Comment:
I think this is a change in behaviour.  We should be initialising the e-stop state based on the switch's position at startup.

Reply:
your right, good catch, I also left it in the copter init function, will fix this evening

Reply:
fixed